### PR TITLE
cancel registration

### PR DESCRIPTION
**Pull request by tlj4k8**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -9,16 +9,15 @@ import VeeValidate from 'vee-validate';
...
 import VueSession from 'vue-session';
 
-
 Vue.use(ElementUI, { locale })
 Vue.config.productionTip = false;
 Vue.use(VeeValidate);
 Vue.use(VueSession);
 Vue.prototype.$axiosServer = axios.create({
-  baseURL: 'https://slalom-health-api-staging.herokuapp.com',
+  baseURL: 'http://localhost:3000',
```
Comment (dborstelmann) [inline src/main.js]: ???

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "axios": "^0.18.0",
     "element-ui": "^2.4.1",
     "express": "^4.16.3",
+    "js-cookie": "^2.2.0",
```
Comment (dborstelmann) [inline package.json]: ???

File: src/main.js (hunk 1 above)
Comment (EricMason24) [reply src/main.js]: cancel registration uses a new route: auth/delete so we need to use local

File: package.json (hunk 2 above)
Comment (EricMason24) [reply package.json]: a possible fix into the csrf issue we were having

File: src/main.js (hunk 1 above)
Comment (EricMason24) [reply src/main.js]: switching it back now

**Pull request merged**